<launch>
    <!-- obj detection node --> 
    <param name="detect_thresh" type="double" value=".4" />
    <node pkg="follow_the_leader" type="detection_node" name="detection_node"  output="screen" />
    <!-- param name="mav_name" type="string" value="$(arg mav_name)" / -->
</launch>
